### PR TITLE
Event Registration: Create QR code locally without server and update QR code data structure

### PR DESCRIPTION
**Pull request by mlenkeit**
This PR updates the document about event registration to represent the latest changes:
- QR codes are generated locally without any server communication.
- The signature is replaced by an alternate approach to calculate the id of a QR code.
- The Protocol Buffer messages are updated accordingly

### CODE REVIEW

File: event_registration.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ Several security and privacy threats have been identified for the proposed solut
 
 ### Profiling of Venues
 
-The proposed solution publishes warnings on CDN. A warning consists of the GUID of a venue and a time interval. An adversary can collect these warnings and aggregate them to compile a list of venues with the most warnings (colloquially referred to as _most infectious venues_) or a list of venues with their most recent warning.
+The proposed solution publishes warnings on CDN. A warning consists of the hashed ID of a venue and a time interval. An adversary can collect these warnings and aggregate them to compile a list of venues with the most warnings (colloquially referred to as _most infectious venues_) or a list of venues with their most recent warning.
 
 This information is easy to collect, as warnings are publicly accessible and do not even required to make modifications to the CWA client.
```
Comment (MikeMcC399) [inline event_registration.md:21]: required => require

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I made one typo / grammar suggestion.

Also the term "Tele TAN" is not consistent with the spelling of "teleTAN" in the https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md document.

The privacy legal document https://www.coronawarn.app/assets/documents/cwa-privacy-notice-en.pdf on the other hand refers to it as "TeleTAN" with no space between "Tele" and "TAN".

Maybe use "TeleTAN" instead of "Tele TAN"?

File: event_registration.md (hunk 1 above)
Comment (mlenkeit) [reply event_registration.md]: Thanks 👍 
Fixed with https://github.com/corona-warn-app/cwa-documentation/pull/571/commits/f6e22e349c4a456fc051dd0721b445c4878ebcd9

**Comment by mlenkeit** [comment]
> Maybe use "TeleTAN" instead of "Tele TAN"?

@MikeMcC399  you're right, "with space" was not a valid spelling, see https://github.com/corona-warn-app/cwa-documentation/blob/master/.spelling#L99-L102

Fixed with https://github.com/corona-warn-app/cwa-documentation/pull/571/commits/678d335559ebf1457addcf35d77ea8488d9ed8ba

**Review by thomasaugsten — APPROVED** [review]

**Pull request merged**